tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -52,6 +72,22 @@ public void processNe(SourceFeature sf, FeatureCollector features) {
     if (!sourceLayer.equals("ne_10m_populated_places")) {
       return;
     }
+
+    // Setup for high zoom content
```

Comment:
Can this be moved out of per tile processing into project startup?

Reply:
I think it's a good idea, you said only 8000 places right? I don't want to slow down the small job tests by loading in the whole thing every time

Reply:
The current config doesn't seem to noticeably affect build time so it's probably OK as is?

But it's better architecture to only load it once on project init. We can file a followup issue for that?

Reply:
Followup in https://github.com/protomaps/basemaps/issues/108.